### PR TITLE
Update data_sources.md

### PR DESCRIPTION
**Pull request by jaluebbe**
Removed data sources not being available anymore to the public. Added airport EDDG/FMO .

### CODE REVIEW

File: _pages/data_sources.md [hunk 1]
```diff
@@ -104,16 +104,14 @@ _(in alphabetic order)_
...
+  - [Münster/Osnabrück](https://opendata.stadt-muenster.de/dataset/flugplandaten-des-flughafen-m%C3%BCnsterosnabr%C3%BCck-fmo) airport in Germany;
   - [Hamburg](https://portal.api.hamburg-airport.de/) airport in Germany;
   - [Hong Kong](https://data.gov.hk/en-data/dataset/aahk-team1-flight-info) airport;
   - [Swedavia](https://apideveloper.swedavia.se/) owns and operates the major airports of Sweden;
 
 - Airlines
   - [Air France and KLM](https://developer.airfranceklm.com/);
-  - [British Airways](https://developer.iairgroup.com/british_airways);
   - [Lufthansa](https://developer.lufthansa.com/);
-  - [Singapore Airlines](https://developer.singaporeair.com/docs/Flight_Status/);
```
Comment (xoolive) [inline _pages/data_sources.md:116]: How about https://developer.singaporeair.com/api_product_catalogue for instance?
It doesn't provide the same info maybe?

Comment (jaluebbe) [reply _pages/data_sources.md:116]: I was using the previous version for a long time. Now you are limited to a sandbox of artificial data and get access to real data for business use only.

Comment (jaluebbe) [reply _pages/data_sources.md:116]: Long term use of a low number of request per day without charge is not accepted anymore.

Comment (xoolive) [reply _pages/data_sources.md:116]: ok!

**Pull request merged**